contracts/scripts/defender-actions/rollup.config.cjs

```diff
@@ -53,9 +52,9 @@ module.exports = [
   },
   {
     ...commonConfig,
-    input: "doAccounting.js",
+    input: "registerValidators.js",
     output: {
-      file: "dist/doAccounting/index.js",
+      file: "dist/registerValidators/index.js",
```

Comment:
switched order since `visualize` plugin overwrites exports, and this way `registerValidator` action is the one that has dependency graphs available